<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnergyConfiguration</class>
 <widget class="QWidget" name="EnergyConfiguration">
 <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>613</width>
      <height>508</height>
     </rect>
    </property>
    <property name="windowTitle">
     <string>Energy configuration</string>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>441</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>  Energy configuration  </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="11" column="1">
         <widget class="QSlider" name="slider_th7">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="10" column="3">
         <widget class="QLabel" name="label_actualEnergies_th6">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLabel" name="label_actualEnergies_th1">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="11" column="3">
         <widget class="QLabel" name="label_actualEnergies_th7">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLabel" name="label_actualEnergies_th2">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QSlider" name="slider_th5">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_th5">
          <property name="text">
           <string>Threshold 5</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_th4">
          <property name="text">
           <string>Threshold 4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="spinBox_th0">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> keV</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>4.000000000000000</double>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QLabel" name="label_actualEnergies_th3">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QSlider" name="slider_th6">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="3">
         <widget class="QLabel" name="label_actualEnergies_th5">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QDoubleSpinBox" name="spinBox_th1">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> keV</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>4.000000000000000</double>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QLabel" name="label_actualEnergies_th4">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QDoubleSpinBox" name="spinBox_th3">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> keV</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>4.000000000000000</double>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_actualEnergies_th0">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSlider" name="slider_th2">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="slider_th0">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_th6">
          <property name="text">
           <string>Threshold 6</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QDoubleSpinBox" name="spinBox_th2">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> keV</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>4.000000000000000</double>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QDoubleSpinBox" name="spinBox_th6">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> keV</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>4.000000000000000</double>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QDoubleSpinBox" name="spinBox_th7">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> keV</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>4.000000000000000</double>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_th3">
          <property name="text">
           <string>Threshold 3</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_th1">
          <property name="text">
           <string>Threshold 1</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QSlider" name="slider_th4">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSlider" name="slider_th1">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QDoubleSpinBox" name="spinBox_th5">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> keV</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>4.000000000000000</double>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QSlider" name="slider_th3">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_th2">
          <property name="text">
           <string>Threshold 2</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_th7">
          <property name="text">
           <string>Threshold 7</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_th0">
          <property name="text">
           <string>Threshold 0</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QDoubleSpinBox" name="spinBox_th4">
         <property name="enabled">
          <bool>true</bool>
         </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> keV</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>4.000000000000000</double>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_targetEnergies">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Target</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_actualEnergies">
          <property name="text">
           <string>Actual</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
 </widget>
</ui>
